server.js

```diff
@@ -3,6 +3,7 @@ require('dotenv').config()
 const express = require('express')
 const app = express()
 const mongoose = require('mongoose')
+const student = require('./api/student')
 
 mongoose.connect('mongodb+srv://Pegasusx10:pegasus123@cluster0.749smlf.mongodb.net/student-database?retryWrites=true&w=majority', { useUnifiedTopology: true, useNewUrlParser: true })
```

Comment:
urls should not be hardcoded. They should be picked from an environment config file. database connection should be done separately and that connection should be reused in all files